packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -884,7 +884,12 @@ const DatePickerInput = (props) => {
       </div>
       {!useExternalFormatMask && (
         <div id={formatDescriptionId} className={cx('format-text')}>
-          <VisuallyHiddenText aria-live={DateUtil.isMac() ? 'polite' : 'off'} text={`${invalidEntry} ${intl.formatMessage({ id: 'Terra.datePicker.dateFormatLabel' })} ${format} ${inputDate || ''} `} />
+          <VisuallyHiddenText
+            aria-live={DateUtil.isMac() ? 'polite' : 'off'}
```

Comment:
We should unit test this change

Reply:
This would just be an update to the response for screen reader and wouldn't require specific test.